src/AbstractUpdateAction.php

```diff
@@ -307,9 +307,9 @@ public function hasRetryOnConflict()
      */
     public function setRefresh($refresh = true)
     {
-        \is_bool($refresh) && $refresh = $refresh
-            ? Reindex::REFRESH_TRUE
-            : Reindex::REFRESH_FALSE;
+        if (\is_bool($refresh)) {
+            $refresh = $refresh ? Reindex::REFRESH_TRUE : Reindex::REFRESH_FALSE;
```

Comment:
We should try with something like `@param Reindex::REFRESH_*|bool $refresh`, as the method signature.
can you check it it works? :)

Reply:
yeah sure, I've used `@phpstan-param` because maybe some tools/IDEs do not understand that param.